core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -920,6 +920,25 @@ public void removeDir(StorageDir dir) {
     }
   }
 
+  /**
+   * Get the block store location based on a storage medium.
+   *
+   * @param mediumName storage medium
+   * @return the storage location
+   */
+  public BlockStoreLocation findLocationByMedium(String mediumName) {
```

Comment:
Can't multiple block store locations provide the same medium type, we should return a set here?

Reply:
actually adding it to the `BlockStoreLocation` removes the need for these two functions